gbpservice/nfp/config_orchestrator/agent/loadbalancer.py

```diff
@@ -209,6 +215,16 @@ def create_vip(self, context, vip):
         nfp_logging.clear_logging_context()
 
     @log_helpers.log_method_call
+    def update_vip(self, context, old_vip, vip):
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(vip["description"])
+        nfp_logging.store_logging_context(meta_id=nf_id)
+        nf = common.get_network_function_details(context, nf_id)
+        self._put(context, vip['tenant_id'], 'vip', nf, olf_vip=old_vip,
+                  vip=vip)
+        nfp_logging.clear_logging_context()
```

Comment:
_post, _put, _delete are just to send rest api, we should not add anything to it related to logging stuff. We can write a common function where code is repetitive. But to do it will have to touch other apis also. Do we want to do it now ?